epub33/core/index.html

```diff
@@ -6423,19 +6430,23 @@ <h4>Obfuscation Algorithm</h4>
 				</section>
 
 				<section id="obfus-specifying">
-					<h4>Specifying Obfuscated Resources</h4>
+					<h4>Specifying Obfuscated Fonts</h4>
 
-					<p>Although not technically encrypted data, all obfuscated resources MUST have an entry in the <code
+					<p>Although not technically encrypted data, all obfuscated fonts MUST have an entry in the <code
 							class="filename">encryption.xml</code> file accompanying the EPUB Publication (see <a
 							href="#sec-container-metainf-encryption.xml"></a>).</p>
 
-					<p>EPUB Creators MUST specify an <code>EncryptedData</code> element for each obfuscated resource.
-						Each <code>EncryptedData</code> element MUST contain a child <code>EncryptionMethod</code>
+					<p>EPUB Creators MUST specify an <code>EncryptedData</code> element for each obfuscated font. Each
+							<code>EncryptedData</code> element MUST contain a child <code>EncryptionMethod</code>
 						element whose <code>Algorithm</code> attribute has the value
 							<code>http://www.idpf.org/2008/embedding</code>. The presence of this attribute signals the
-						use of the algorithm described in this specification. EPUB Creators MUST list the path to the
-						obfuscated resource in the <code>CipherReference</code> child of the <code>CipherData</code>
-						element.</p>
+						use of the algorithm described in this specification.</p>
+
+					<p>EPUB Creators MUST list the path to the obfuscated font in the <code>CipherReference</code> child
+						of the <code>CipherData</code> element. Note that EPUB Creators MAY only use the obfuscation
+						algorithm with <a href="#cmt-grp-font">Font Core Media Types</a>. Consequently, the
```

Comment:
Shouldn't we use _SHOULD_ here instead of _MAY_? (It may require a slight rewording of the sentence, though).

Reply:
Oops, that wasn't supposed to be uppercase as it was only intended to be explanatory. Making it a should would contradict the next sentence.

Maybe this can just be condensed down to a single statement:

> As the obfuscation algorithm is restricted to fonts, the <code>URI</code> attribute of the `CipherReference` element MUST only reference Font Core Media Types.

Reply:
It might be good to note this restriction in the intro, too, just to make it extra clear. I'll commit some changes in a minute.